examples/nats/nats_test.go

```diff
@@ -0,0 +1,66 @@
+package nats
+
+import (
+	"context"
+	"testing"
+
+	"github.com/nats-io/nats.go"
```

Comment:
Could you run `go mod tidy` in the root dir of the nats example module? I think we are missing the synced modules